cmd/sealer/cmd/cluster/run.go

```diff
@@ -73,6 +77,43 @@ func NewRunCmd() *cobra.Command {
 				clusterFile     = runFlags.ClusterFile
 				applyMode       = runFlags.Mode
 			)
+			imageEngine, err := imageengine.NewImageEngine(imagecommon.EngineGlobalConfigurations{})
+			if err != nil {
+				return err
+			}
+
+			extension, err := imageEngine.GetSealerImageExtension(&imagecommon.GetImageAnnoOptions{ImageNameOrID: args[0]})
+			if err != nil {
+				return fmt.Errorf("failed to get cluster image extension: %s", err)
+			}
+
+			if extension.Type == v12.AppInstaller {
+				logrus.Infof("start to install application: %s", args[0])
```

Comment:
the args[0] is not a application name, how about use  `start to install app image` here